core/codegen/src/attribute/route/parse.rs

```diff
@@ -29,6 +30,8 @@ pub struct Route {
     pub handler: syn::ItemFn,
     /// The parsed arguments to the user's function.
     pub arguments: Arguments,
+    /// The doc comment describing this route
+    pub doc_comment: Option<String>
```

Comment:
```suggestion
    pub docstring: String,
```

Should this be `Option`? I think an empty `String` would be fine.